index.js

```diff
@@ -14,6 +14,14 @@ var VersionChecker = require('ember-cli-version-checker');
 module.exports = {
   name: 'Ember CLI Mocha',
 
+  shouldIncludeChildAddon: function(childAddon) {
+    if (childAddon.name === 'ember-cli-test-loader') {
+      return true;
+    } else {
+      return this._super.shouldIncludeChildAddon.apply(this, arguments);
+    }
+  },
```

Comment:
@alexlafroscia can you explain why this hook is needed? it seems like the default implementation returns `true` in all cases and this hook is mainly meant to filter addons *out* instead of explicitly including them.